### DISCUSSION TITLE
Tshock won't start

### DISCUSSION THREAD

**Discussion by on1ion2** [Pryaxis/TShock#2881] [General]
Hi, i was just about to start a new terraria server with my brother but when i run "Tshock.server" it seems to not be working it says something i captured in the video at some point. btw i already installed .net6.0 but the problem remains

https://user-images.githubusercontent.com/105763956/208348373-1d9c6054-2cf3-47d4-a7c9-cf73dd8f5394.mp4

**Comment by on1ion2** [comment]
i had played on the older version before(1.4.3) and it works pretty fine

**Comment by sgkoishi** [comment]
Try right click the white space in the folder.
Choose `Open PowerShell here`, type `./TShock.Server.exe` and press enter and it should keep the error message.
If there's no PowerShell option, choose `Open Command Prompt here` and type `TShock.Server.exe`.

Or, the [latest dev version](https://github.com/Pryaxis/TShock/actions/workflows/ci-otapi3.yml?query=event%3Apush+branch%3Ageneral-devel) contains an installer and you can try it.

**Reply by on1ion2** [reply]
thanks i'll try!

**Reply by on1ion2** [reply]
![image](https://user-images.githubusercontent.com/105763956/208354364-7abaab2d-82af-4ed0-b1e7-868ce7e21a7a.png)

**Reply by sgkoishi** [reply]
As it said, it can't find your dotnet. The latest dev version contains a `TShock.Installer` and you can run it instead—it will setup and launch TShock itself.

**Reply by on1ion2** [reply]
actually, how do i run the installer since it's not an exe file. Or i just didn't find the right version

**Reply by sgkoishi** [reply]
From the link you can find many workflow runs. Click the first one with `CI(OTAPI3)` (for now it is [this one](https://github.com/Pryaxis/TShock/actions/runs/3683343469)), scroll down and find your version (you are using Windows, so you want the last one `TShock-Beta-win-x64-Release`). There is a `TShock.Installer.exe` inside.

**Reply by on1ion2** [reply]
Thanks alot
